lib/tailwind_formatter.ex

```diff
@@ -18,11 +18,13 @@ defmodule TailwindFormatter do
   def format(contents, _opts) do
     contents
     |> HEExTokenizer.tokenize()
-    |> Enum.reduce(contents, fn
+    |> Enum.reduce([contents], fn
       {:tag, _name, attrs, _meta}, contents ->
         Enum.reduce(attrs, contents, fn
-          {"class", class, _meta}, contents ->
-            sort_classes(class, contents)
+          {"class", class, _meta}, [remainder | acc] ->
+            {old, sorted} = sort_classes(class)
+            [attr, remainder] = String.split(remainder, old, parts: 2)
+            [remainder, sorted, attr | acc]
```

Comment:
What do you think about `sort_classes/1` returning only the sorted classes, instead of a tuple? Then:

```suggestion
            [attr, remainder] = String.split(remainder, class, parts: 2)
            [remainder, sort_classes(class), attr | acc]
```

Reply:
It does look cleaner to separate the extraction 👍 I went with the tuple approach because of the shape of `class` being of `{:type, classes, meta}`. Can solve it by creating a `old_classes` function that fetches the class string